session/session.go

```diff
@@ -1624,8 +1624,11 @@ func (s *session) ClearDiskFullOpt() {
 func (s *session) ExecuteInternal(ctx context.Context, sql string, args ...interface{}) (rs sqlexec.RecordSet, err error) {
 	origin := s.sessionVars.InRestrictedSQL
 	s.sessionVars.InRestrictedSQL = true
+	oriSQLMode := s.sessionVars.SQLMode
+	s.sessionVars.SQLMode = mysql.DelSQLMode(oriSQLMode, mysql.ModeNoBackslashEscapes)
```

Comment:
Why not just set it to default sql mode?

Reply:
Got it. Some function like `checkColumnDefaultValue` depends on `sql_mode`, which will be called by `ExecuteInternal` during bootstrap. If we want to use the default sql_mode, we'll need to review all bootstrap sqls and check whether they can work under the default sql_mode? So, I think it's fine to use `DelSQLMode` :beers: . Good job!